src/xenia/app/emulator_window.cc

```diff
@@ -39,6 +39,10 @@
 #include "xenia/ui/ui_event.h"
 #include "xenia/ui/virtual_key.h"
 
+#ifdef XE_PLATFORM_LINUX
```

Comment:
I suspect that this whole change should be placed somewhere in xenia/ui/window_gtk.cc